File: 1100/2009C/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;

    while (t--) {
        long long x, y, k;
        cin >> x >> y >> k;

        long long moves_x = x / k + (x % k == 0 ? 0 : 1);
        long long moves_y = y / k + (y % k == 0 ? 0 : 1);

        long long cnt = max(2 * moves_x - 1, 2 * moves_y);

        cout << cnt << "\n";
    }

    return 0;
}

File: 1100/2009C/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            long x = sc.nextLong();
            long y = sc.nextLong();
            long k = sc.nextLong();
            
            long cnt = 0;
            
            long moves_x = x/k + (x%k == 0 ? 0:1);
            long moves_y = y/k + (y%k == 0 ? 0:1);
            
            cnt += Math.max(2*moves_x-1, 2*moves_y);
            
            System.out.println(cnt);
        }
    }
}